src/schedule/2021-01-14-data-fetching-with-server-components.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Data Fetching With Server Components
+type: Episode Party
+date: 2021-01-14
+layout: layouts/event.html
```

Comment:
We're already setting the layout for all of our events by default! This happens in [`src/schedule/schedule.json`](https://github.com/LunchDevCommunity/community-calendar/tree/main/src/schedule), which is a [directory data file](https://www.11ty.dev/docs/data-template-dir/).

Because of that, I propose we leave the `layout` property off here, and only include it when we need to override the defaults. What do you think?

Reply:
aaah. yes. i copy/pasted this from another event that didn't get the memo either. will pull it from all events.